src/mcts/search.cc

```diff
@@ -1313,7 +1314,10 @@ void SearchWorker::DoBackupUpdateSingleNode(
       v = n->GetQ();
       d = n->GetD();
     }
-    n->FinalizeScoreUpdate(v, d, node_to_process.multivisit);
+    n->FinalizeScoreUpdate(
+		v * std::max(0.0f, 1.0f - params_.GetShortSightedness() * depth),
```

Comment:
If `depth = 0`, this returns `v * 1`, but for `depth >= 1` this gives `v * 0` since `1 - depth <= 0`.

Reply:
err shortsightedness is a float parameter with a maximum possible value of 0.03.  there is no calculation of 1 - depth.

Reply:
Ha. For some reason I thought it was a boolean.

Reply:
There are definitely other options for decay. v / (1 + param * depth) for instance would avoid the need to clamp without ever leaving [-1,1] bounds.  But will start with this simple one for some testing to start.  I need to set up a 'vs weaker opponent' test to see if it has a bias towards that scenario.

Reply:
Switched to 1 / ( 1 + param * depth) - seems to work close enough to before, just with a slightly increased parameter value.